Exercise_A/index.html

```diff
@@ -20,19 +27,135 @@
   </head>
 
   <!-- Nav bar setup -->
+ 
 
   <body data-spy="scroll" data-target=".navbar">
     <!-- data-spy="scroll" data-target=".navbar" data-offset="50" -->
+
     <!--Allows us to track the progress of the scroll -->
+
     <!-- Nav bar items -->

```

Comment:
The reason the buttons on you navigation bar do not work is because you need to assign the id to the element that you're trying to scroll too. Look at comments below. 

Also you have some link buttons kind of just shoved to the right of the navigation bar.  You can probably delete them, lines 58-70